week-2/implement/cases.js

```diff
@@ -13,3 +13,22 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+
+
+function toUpperCamelCase(inputString) {
+    // Dividir a string em palavras usando espaços como delimitadores
```

Comment:
this whole function could be simplified into `return inputString.toUpperCase().replaceAll(" ", "_")`